src/applications/model/gaming-traffic-model.cc

```diff
@@ -99,7 +105,8 @@ MixtureRandomVariable::IsValid (std::vector<double> probs)
     {
       cumProb += prob;
     }
-  if (fabs (cumProb - 1) < std::numeric_limits<double>::epsilon ())
+  // check if cumProb == 1 +/- epsilon (1e-9)
+  if (fabs (cumProb - 1) < 1e-9)
```

Comment:
ok, let's remember, though, that when `cumProb` will be saved as a member variable, it will have to be normalize to 1, and possible the last element should be set manually to one to avoid the `NS_FATAL_ERROR `in `GetValue`